### ISSUE TITLE
[Suggestion] Ability to rename bots instead of e-mail.

### ISSUE THREAD

**Issue by jstgnkl** [OpenPoGo/OpenPoGoWeb#135]
Would love to be able to change the name of my bots. So at least people who could access my web page won't be able to instantly see my e-mail.

**Comment by jadbalout** [comment]
I'm thinking of ways to do this, maybe changing the users array into a dictionary like so:

```
users: {"xd@gmail.com":"USERNAME_I_WANT",
"xd2@gmail.com":"Cool Bot"}
```

What do you think?

**Comment by BobbyWibowo** [comment]
@jadbalout To be fair, the config file is in a JS file which will be loaded alongside everything else, so it's still possible to peek the actual username.

**Comment by jadbalout** [comment]
@BobbyWibowo Fair enough. I guess that can't be done without removing the users and placing them in SQL Databases, which we do not intend to do.

**Comment by BobbyWibowo** [comment]
@jadbalout Yeah, but I personally still like a simple display name feature. I have done this on my personal repo. I roughly implemented it similar with your idea.

**Comment by terbolous** [comment]
Is there any other usage of this value, except picking the correct json files? If not then you have a couple of options, you could for instance symlink (or hardlink if you prefer) 'inventory-myusername@email.com.json' to 'inventory-CoolBot.json'.

Or ask for a similar feature in the bot (a config value for setting the json filenames)

**Comment by BobbyWibowo** [comment]
@eriweb As far as I'm aware, it's indeed only for getting the correct json files.
